uint32 seq	    # sequence ID: consecutively increasing ID
time stamp	    # Two-integer timestamp of data
string agent_id    # Robot Data is associated with
float64 x1	    # X-coordinate of robot
float64 y1	    # Y-coordinate of robot
float64 yaw1       # Yaw (heading) of robot
float64 x2	    # X-coordinate of robot
float64 y2	    # Y-coordinate of robot
float64 yaw2       # Yaw (heading) of robot
